package.json

```diff
@@ -83,6 +83,5 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  },
-  "packageManager": "yarn@4.2.2"
```

Comment:
@0x4007 Don't we want this? It can be `1.22` but setting it avoids collaborator to use the wrong package manager and encounter strange errors.

Reply:
It was unintentional, reverted it